src/components/RegistrationForm/RegistrationForm.styled.js

```diff
@@ -7,7 +7,6 @@ import { device } from '../../utils';
  */
 const StyledFlexDiv = styled.div`
   display: flex;
-  // position: ${props => props.$position || 'static'};
   flex-direction: ${props => props.$direction || 'column'};
```

Comment:
We can delete all these props and change they to basic css.

Reply:
ok!